team_production_system/models.py

```diff
@@ -258,6 +258,42 @@ def mentee_cancel_notify(self):
             recipient_list=[self.mentee.user.email],
         )
 
+    # Notify a user that a session is coming up in 60 min
+    def sixty_min_notify(self):
```

Comment:
Let's add a comment here for unit testing. Ex:` #TODO: Needs unit testing`

Reply:
Added comment